src/libraries/System.Security.Cryptography.Cose/tests/CoseTestSign1.cs

```diff
@@ -0,0 +1,121 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+using Xunit;
+
+namespace System.Security.Cryptography.Cose.Tests
+{
+    public sealed class CoseTestSign1
```

Comment:
Similarly, why do we have a new class with similarly named responsibility as `CoseSign1MessageTests.*`?